eng/pipelines/runtimelab.yml

```diff
@@ -6,7 +6,7 @@ trigger:
   batch: true
   branches:
     include:
-    # Add Experiment branch here
+    - FreeBSD
```

Comment:
This should be `feature/FreeBSD`

Reply:
ok. I followed the linked example https://github.com/dotnet/runtimelab/pull/25/files

Reply:
updated

Reply:
We have added `feature/` prefix to all branches after this example was created. I have merged a PR that enabled the CI for this branch (without the rest of the changes in this PR), and update the example to point to it.

Sorry about the merge conflict that you may get.